bitnami/moodle/README.md

```diff
@@ -265,6 +266,46 @@ You may want to review the [PV reclaim policy](https://kubernetes.io/docs/tasks/
 
 ## Upgrading
 
+### To 9.0.0
+
+MariaDB dependency version was bumped to a new major version that introduces several incompatilibites. Therefore, backwards compatibility is not guaranteed unless an external database is used. Check [MariaDB Upgrading Notes](https://github.com/bitnami/charts/tree/master/bitnami/mariadb#to-800) for more information.
+
+To upgrade to `9.0.0`, it should be done reusing the PVCs used to hold both the MariaDB and Moodle data on your previous release. To do so, follow the instructions below (the following example assumes that the release name is `moodle`):
+
+> NOTE: Please, create a backup of your database before running any of those actions. The steps below would be only valid if your application (e.g. any plugins or custom code) is compatible with MariaDB 10.5.x
+
+Obtain the credentials and the names of the PVCs used to hold both the MariaDB and Moodle data on your current release:
+
+```console
+export MOODLE_PASSWORD=$(kubectl get secret --namespace default moodle -o jsonpath="{.data.moodle-password}" | base64 --decode)
+export MARIADB_ROOT_PASSWORD=$(kubectl get secret --namespace default moodle-mariadb -o jsonpath="{.data.mariadb-root-password}" | base64 --decode)
+export MARIADB_PASSWORD=$(kubectl get secret --namespace default moodle-mariadb -o jsonpath="{.data.mariadb-password}" | base64 --decode)
+export MARIADB_PVC=$(kubectl get pvc -l app.kubernetes.io/instance=moodle,app.kubernetes.io/name=mariadb,app.kubernetes.io/component=primary -o jsonpath="{.items[0].metadata.name}")
+export MOODLEDATA_PVC=$(kubectl get pvc -l app.kubernetes.io/instance=moodle,app.kubernetes.io/name=moodle -o jsonpath="{.items[0].metadata.name}")
```

Comment:
I would say this is not needed since you will be setting `replicaCount` to 0 and the PVC will be reused when upgrading to the new version of the chart. Let's check it, please.

Reply:
Yes, it is indeed not needed. It is removed now.